staging/src/k8s.io/kubectl/pkg/cmd/config/rename_context.go

```diff
@@ -63,7 +64,7 @@ func NewCmdConfigRenameContext(out io.Writer, configAccess clientcmd.ConfigAcces
 	cmd := &cobra.Command{
 		Use:                   renameContextUse,
 		DisableFlagsInUseLine: true,
-		Short:                 renameContextShort,
+		Short:                 i18n.T(renameContextShort),
```

Comment:
Variables cannot be translated. I think the string inside this variable needs to be translated

Reply:
Yeah, you should use `i18n.T` on the string in `renameContextShort` variable text.